Bug: intermittent failure search test failure

<!-- Bug or Feature Pick one and start your tile with it. --> 

## Expected behavior
All test cases should pass consistently. Increase timeouts to prevent error.

## Actual behavior
occasionally the following error occurs.
```
======================================================================
FAIL: test_search_returns_N_results_when_N_documents_match_query (test_search.TestAWSSearch) [Search Returns 10001 Matches When 10001 Documents Indexed]
Create N identical documents. A search query is executed to match the document. All documents created must be
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/HumanCellAtlas/data-store/tests/test_search.py", line 146, in test_search_returns_N_results_when_N_documents_match_query
    self.verify_search_results(smartseq2_paired_ends_query, x, bundle_ids)
  File "/home/travis/build/HumanCellAtlas/data-store/tests/test_search.py", line 203, in verify_search_results
    self.fail("elasticsearch failed to return all results.")
AssertionError: elasticsearch failed to return all results.
----------------------------------------------------------------------
```
This is the travis build it occured: https://travis-ci.org/HumanCellAtlas/data-store/builds/282467010?utm_source=email&utm_medium=notification

--- a/tests/test_indexer.py
+++ b/tests/test_indexer.py
@@ -246,7 +246,7 @@ class TestIndexerBase(DSSAsserts, StorageTestSupport, DSSUploadMixin):
 
     @staticmethod
     def get_notification_payload():
-        timeout = 2
+        timeout = 5
         timeout_time = time.time() + timeout
         while True:
             posted_payload_string = PostTestHandler.get_payload()
@@ -333,7 +333,7 @@ class TestIndexerBase(DSSAsserts, StorageTestSupport, DSSUploadMixin):
         # By default, the refresh interval is one second.
         # https://www.elastic.co/guide/en/elasticsearch/reference/5.5/_modifying_your_data.html
         # Yet, sometimes one second is not quite enough given the churn in the local Elasticsearch instance.
-        timeout = 2
+        timeout = 5
         timeout_time = time.time() + timeout
         while True:
             response = ElasticsearchClient.get(logger).search(

--- a/tests/test_search.py
+++ b/tests/test_search.py
@@ -188,7 +188,7 @@ class TestSearchBase(DSSAsserts):
             json_request_body=dict(es_query=es_query),
             expected_code=status_code)
 
-    def verify_search_results(self, es_query, expected_result_length=0, bundles=[], timeout=5):
+    def verify_search_results(self, es_query, expected_result_length=0, bundles=[], timeout=8):
         timeout_time = timeout + time.time()
         while time.time() <= timeout_time:
             response = self.post_search(es_query, requests.codes.ok)

